tests/unit/Lookup/UnknownForeignRepositoryExceptionTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Tests\Lookup;
+
+use Exception;
+use Wikibase\DataModel\Services\Lookup\UnknownForeignRepositoryException;
+
+/**
+ * @covers Wikibase\DataModel\Services\Lookup\UnknownForeignRepositoryException
+ *
+ * @license GPL-2.0+
+ */
+class UnknownForeignRepositoryExceptionTest extends \PHPUnit_Framework_TestCase {
+
+	public function testConstructWithRepositoryNameOnly() {
+		$exception = new UnknownForeignRepositoryException( 'foo' );
+
+		$this->assertSame( 'foo', $exception->getRepoName() );
+		$this->assertSame( 'Unknown repository name: foo', $exception->getMessage() );
+		$this->assertSame( 0, $exception->getCode() );
+		$this->assertNull( $exception->getPrevious() );
+	}
+
+	public function testConstructWithAllArguments() {
+		$previous = new Exception( 'Dummy exception' );
```

Comment:
The dummy string is not necessary, it does not add value to the test. It's even a bit confusing to call this a "dummy", because it is none. Could be removed.